fe/fe-core/src/main/java/org/apache/doris/analysis/CastExpr.java

```diff
@@ -142,11 +142,6 @@ private static boolean disableRegisterCastingFunction(Type fromType, Type toType
                 || toType.isDateType())) {
             return true;
         }
-
```

Comment:
Here is the back door is not just for spark load. All of the castExpr to bitmap are allowed.
I think we'd better figure out why the object store type was forbidden to be converted before, and which type conversions are definitely safe